src/V3Param.cpp

```diff
@@ -758,68 +758,94 @@ class ParamProcessor final {
         }
     }
 
-public:
-    void cellDeparam(AstCell* nodep, AstNodeModule* modp, const string& someInstanceName) {
-        m_modp = modp;
-        // Cell: Check for parameters in the instantiation.
-        // We always run this, even if no parameters, as need to look for interfaces,
-        // and remove any recursive references
-        UINFO(4, "De-parameterize: " << nodep << endl);
-        // Create new module name with _'s between the constants
-        if (debug() >= 10) nodep->dumpTree(cout, "-cell: ");
-        // Evaluate all module constants
-        V3Const::constifyParamsEdit(nodep);
-        AstNodeModule* const srcModp = nodep->modp();
-        srcModp->someInstanceName(someInstanceName + "." + nodep->name());
-
+    bool nodeDeparamCommon(AstNode* nodep, AstNodeModule*& srcModp, AstPin* paramsp, AstPin* pinsp,
```

Comment:
```suggestion
    bool nodeDeparamCommon(AstNode* nodep, AstNodeModule*& srcModpr, AstPin* paramsp, AstPin* pinsp,
```

pr = reference to a pointer.  Similar renames please where added other non-const references.